main/ZgatewayWeatherStation.ino

```diff
@@ -105,7 +105,9 @@ void sendTemperatureData(byte id, float temperature, int humidity, byte battery_
     StaticJsonBuffer<JSON_MSG_CALC_BUFFER> jsonBuffer;
     JsonObject& RFdata = jsonBuffer.createObject();
     RFdata.set("sensor", id);
-    RFdata.set("temperature", temperature);
+    RFdata.set("temperature", temperature); // remove for 0.9.6 release
+    RFdata.set("tempc", temperature);
+    RFdata.set("tempf", wsdr.convertCtoF(temperature));
```

Comment:
why is here another conversion function used?

Reply:
When a library supports its own conversion method, I used it. Otherwise I used the conversion macros in user_config.h